op-challenger/init_game.sh

```diff
@@ -4,8 +4,7 @@ set -euo pipefail
 DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
 cd "$DIR"
 
-# build the challenger to keep it up to date
-make
+make clean && make
```

Comment:
We probably shouldn't need to `make clean` every time. `make` generally should manage to update anything that changed for us.